shared.js

```diff
@@ -148,6 +148,33 @@ module.exports = (Channel, slack) => {
             }).exec();
         },
 
+        isManagedChannel: async function(channel_id) {
+            return Channel.findById(channel_id).exec();
+        },
+
+        removeUserFromChannel: async function(channel, user) {
+            try {
+                const res = await slack.user.conversations.kick({
```

Comment:
Have we tested this with the bot to ensure it has the needed scopes?

Reply:
tested with the test bot that has similar scopes, but not exactly the same.